website/docs/DriverBinaries.md

```diff
@@ -3,7 +3,59 @@ id: driverbinaries
 title: Driver Binaries
 ---
 
-To run automation based on the WebDriver protocol you need to have browser drivers set up that translate the automation commands and are able execute them in the browser. With WebdriverIO `v8.14` and above there is no need to manually download and setup any browser drivers anymore as this is handled by WebdriverIO. You only need to have a browser installed, that's it!
+To run automation based on the WebDriver protocol you need to have browser drivers set up that translate the automation commands and are able to execute them in the browser.
+
+## Automated setup
+
+With WebdriverIO `v8.14` and above there is no need to manually download and setup any browser drivers anymore as this is handled by WebdriverIO. All you have to do is specify the browser you want to test and WebdriverIO will do the rest. For more information on this feature and its limitations, see [here](capabilities#automate-different-browser-channels).
+
+## Manual setup
+
+If the automated setup is not suitable for your use case (e.g. complex network rules that limit Internet access), you can still manually setup the drivers. All you have to do is to specify the driver binary and WebdriverIO will skip the automated download. For example:
+
+```ts
+{
+    capabilities: [
+        {
+            browserName: 'chrome', // or 'firefox', 'msedge'
+            'wdio:chromedriverOptions': { // or 'wdio:geckodriverOptions', 'wdio:edgedriverOptions'
+                binary: '/path/to/chromedriver' // or 'geckodriver', 'msedgedriver'
+            }
+        }
+    ]
+}
+```
+
+The previous configuration will indicate WebdriverIO to automatically start/stop the specified `chromedriver` binary using an arbitrary unused port. If you need more control over the driver, simply specify a value for [port](configuration#port) and WebdriverIO will skip the automated start/stop. For example:
```

Comment:
I am not sure if it is valuable to document this. You example works for a single session but it becomes a lot more complex when multiple sessions are involved. Furthermore you example does the same as when using WebdriverIOs built-in setup because Geckodriver is downloaded when you call `start` so it will fail without network connections too.

Reply:
> I am not sure if it is valuable to document this.

Agreed, it's very use-case specific, and the `## Manual setup` provides enough information on how to get/start the drivers.

> Furthermore you example does the same as when using WebdriverIOs built-in setup because Geckodriver is downloaded when you call `start` so it will fail without network connections too.

Yeah, I missed to specify the `customGeckoDriverPath` parameter, which unfortunately is broken. In other words, this won't work:

```js
{
  browserName: 'firefox',
  'wdio:geckodriverOptions': {
    binary: '/path/to/geckodriver'
  }
}
```

That being said, I have created https://github.com/webdriverio-community/node-geckodriver/pull/222 to fix it.